src/args.rs

```diff
@@ -44,6 +51,13 @@ impl Args {
             no_cache: cli.no_cache,
         }
     }
+
+    fn read_password_from_stdin() -> String {
+        print!("enter password for repository: ");
```

Comment:
We can never print to stdout. This is reserved for printing the exclude list generated from the marked entries so that the user can redirect stdout and get just the marks.

Use stderr instead.